site/content/en/news/releases/notes/v1.2.0.md

```diff
@@ -7,6 +7,7 @@ Date: November 06, 2024
 
 ## Breaking Changes
 - Gateway API GRPCRoute and ReferenceGrant v1alpha2 have been removed
+- The Experimental supportedFeatures field in GatewayClass status has changed from being a list of strings to being a list of objects structs with a name field
```

Comment:
This shouldnt affect EG users, CRDs need to be reapplied during upgrade

Reply:
The existing resources cause EG raise complains. 

```
2024-11-28T12:49:28.144Z	INFO	provider	cache/reflector.go:561	pkg/mod/k8s.io/client-go@v0.31.2/tools/cache/reflector.go:243: failed to list *v1.GatewayClass: json: cannot unmarshal string into Go struct field GatewayClassStatus.items.status.supportedFeatures of type v1.SupportedFeature	{"runner": "provider"}
2024-11-28T12:49:28.144Z	ERROR	provider	cache/reflector.go:158	Unhandled Error	{"runner": "provider", "logger": "UnhandledError", "error": "pkg/mod/k8s.io/client-go@v0.31.2/tools/cache/reflector.go:243: Failed to watch *v1.GatewayClass: failed to list *v1.GatewayClass: json: cannot unmarshal string into Go struct field GatewayClassStatus.items.status.supportedFeatures of type v1.SupportedFeature"}
```

https://envoyproxy.slack.com/archives/C03E6NHLESV/p1732798284104539

Reply:
they were using it with Istio afaik which was populating the previous version